content/docs/2.2/scalers/mongodb.md

```diff
@@ -15,48 +15,85 @@ This specification describes the `mongodb` trigger that scales based on result o
 triggers:
   - type: mongodb
     metadata:
+      # name of an environment variable containing a valid MongoDB connection string
+      connectionStringFromEnv: MongoDB_CONNECTION_STRING
       # Required: database name
       dbName: "test"
       # Required: collection name
       collection: "test_collection"
       # Required: query expr, used by filter data
       query: '{"region":"eu-1","state":"running","plan":"planA"}'
-      # Required: according to the number of query result, to scale job
+      # Required: according to the number of query result, to scale the TargetRef
       queryValue: "1"
-      # Optional: This value will default to a masked version of the host and collection name if not set by the user (metrics name value would be then `mongodb-https---xxx-test_collection`)
```

Comment:
Why was the example of generated metricName deleted? `mongodb-https---xxx-test_collection`
If user doesn't provide a `metricName` value, this one will be generated based on the rest of the values from the example.

Reply:
Thanks @zroubalik  for taking look in this. 
Actually in this example the value of  `metricName` is mentioned, so i used this value for the final generated metricsName which will use in the HPA.
And i also mentioned the case separately if  the user doesn't not specify `metricName` value, then what will happen. in line number 84. 
if you will say, I will add `mongodb-mongodb---xxx-test_collection`  in line 84, for more clarity.

Reply:
Yeah, great. Just add a simple note that if metricName is not set, then one is generated based on the values in trigger spec, with the specific example
 Thanks!

Reply:
Just for clarification where should i add this note? either after the both yaml example or somewhere else? 
because i already mentioned it at line 84 but without specific example.

Thanks!

Reply:
I'd add it on both. thx

Reply:
Hi @zroubalik 
I have addressed the comments, PTAL